src/azure-cli/azure/cli/command_modules/cosmosdb/custom.py

```diff
@@ -247,7 +268,9 @@ def cli_cosmosdb_update(client,
         network_acl_bypass=network_acl_bypass,
         network_acl_bypass_resource_ids=network_acl_bypass_resource_ids,
         api_properties=api_properties,
-        backup_policy=backup_policy)
+        backup_policy=backup_policy,
+        identity=system_assigned_identity,
+        default_identity=default_identity)
```

Comment:
```suggestion
        default_identity=default_identity or existing.default_identity)
```
Should we use the original value as default value for `update` operation?